### PR TITLE
this is a test mario

### PR DESCRIPTION
**Pull request by mariomartinez81**

### CODE REVIEW

**Comment by gabrieldegiorgi** [comment]
It's OK!

**Comment by LeandroCadena** [comment]
nice

File: README.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 Henry app
 test 2
 
-test mario
\ No newline at end of file
+this is a test mario ...
```
Comment (LeandroCadena) [inline README.md:5]: this is ok

File: README.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
 Henry app
 test 2
```
Comment (gabrieldegiorgi) [inline README.md:4]: //Hola Mario!!!

File: README.md (hunk 1 above)
Comment (LeandroCadena) [reply README.md:5]: change mario for Mario

**Review by LeandroCadena — APPROVED** [review]

**Review by gabrieldegiorgi — APPROVED** [review]

**Pull request merged**